definitions/types/k8s-node-taint.json

```diff
@@ -0,0 +1,46 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "type": "object",
+    "title": "Taint",
+    "description": "A taint to apply to Kubernetes nodes",
+    "required": [
+        "taint_key",
+        "taint_value",
+        "effect"
+    ],
+    "properties": {
+        "taint_key": {
+            "$ref": "../types/k8s-resource-name.json",
+            "title": "Key",
+            "examples": [
+                "workloads"
+            ]
+        },
+        "taint_value": {
+            "$ref": "../types/k8s-resource-name.json",
+            "title": "Value",
+            "examples": [
+                "gpu"
+            ]
+        },
+        "effect": {
+            "type": "string",
+            "title": "Effect",
+            "default": "NO_SCHEDULE",
+            "enum": [
```

Comment:
Does `enum` work with `const` and `title` pairs? I thought it was just `oneOf`

Reply:
Great question! I did run this through `mass bundle build` but let me check if this is valid json schema. I want the const / title thingy here. :)

Reply:
Made the fix for `oneOf`, ran that through bundle preview.